src/clients/cli.rs

```diff
@@ -27,6 +28,57 @@ impl Cli {
         self.run_with_args(image, RunArgs::default())
     }
 
+    pub fn run_and_exec<I: ImageExt>(&self, image: I) -> Container<'_, I> {
```

Comment:
Why do we need a separate function for this? I think this should be default behaviour. We should keep the API minimal.